src/metpy/calc/thermo.py

```diff
@@ -359,6 +359,9 @@ def _lcl_iter(p, p0, w, t):
     fp = so.fixed_point(_lcl_iter, pressure.m, args=(pressure.m, w, temperature),
                         xtol=eps, maxiter=max_iters)
     lcl_p = fp * pressure.units
+    if np.isclose(lcl_p, pressure):
```

Comment:
Can you add a comment here so that two years from know we’ll know why we did this?

Reply:
Include the issue # in that comment so we can find it too, please.